deepctr/layers/core.py

```diff
@@ -63,8 +66,13 @@ def build(self, input_shape):
             raise ValueError("Unexpected inputs dimensions %d and %d, expect to be 3 dimensions" % (
                 len(input_shape[0]), len(input_shape[1])))
 
-        if input_shape[0][-1] != input_shape[1][-1] or input_shape[0][1] != 1:
-            raise ValueError('A `LocalActivationUnit` layer requires '
+        if self.self_attention and input_shape[0][-1] != input_shape[1][-1]:
+            raise ValueError('A `LocalActivationUnit` layer with self_attention is True requires '
```

Comment:
"requires inputs of a two inputs with shape" -> "requires two inputs with shape"?

Reply:
Yes, thank you very much ! My English is not very good. I will change the code.